tests/monitoring/metrics.go

```diff
@@ -0,0 +1,150 @@
+package monitoring
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	"github.com/onsi/gomega/types"
+	"github.com/prometheus/client_golang/prometheus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	virtapi "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-api"
+	virtcontroller "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-controller"
+	virtoperator "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-operator"
+
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libmonitoring"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/libwait"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+var _ = Describe("[Serial][sig-monitoring]Metrics", Serial, decorators.SigMonitoring, func() {
```

Comment:
Why do we need to run it serial? or there any cluster-wide changes made as part of these tests?

Reply:
removed